src/ctelnet.cpp

```diff
@@ -1670,18 +1670,18 @@ void cTelnet::processTelnetCommand(const std::string& telnetCommand)
 
         if (option == OPT_MSDP) {
             //MSDP support
-            std::string output;
             if (!mpHost->mEnableMSDP) {
-                output += TN_IAC;
-                output += TN_DONT;
-                output += OPT_MSDP; // disable MSDP per http://tintin.sourceforge.net/msdp/
-                // This will be unaffected by Mud Server encoding:
-                socketOutRaw(output);
-#ifdef DEBUG_TELNET
-                qDebug() << "WE send telnet IAC DONT MSDP";
-#endif
```

Comment:
I think we will still want this - unless it gets reported in the `sendTelnetOption(...)` code {I don't have the source code open in front of me, I'm just looking through what is here!}

Reply:
This gets handled in the function that replaces it and is standardized to the remainder of the code blocks below it.